pkg/ingester/stream.go

```diff
@@ -387,6 +397,7 @@ func (s *stream) validateEntries(entries []logproto.Entry, isReplay bool) ([]log
 		}
 	}
 
+	s.rateCalculator.Record(int64(totalBytes))
```

Comment:
Why is this not using `validBytes`?

Reply:
This preserves the current behavior where stream rate limits are calculated against all the bytes in the stream, not just the valid ones. I assume we do it this way because the rate limit is to protect the ingester and we process every entry, valid or not.